action-scripts/build.js

```diff
@@ -1,246 +1,257 @@
 const StyleDictionary = require("style-dictionary").extend("config.json");
 
-/* ======================= Helpers ======================= */
+const build = () => {
```

Comment:
Error `build is not a function` in `main.js` importing/calling `build()`. I just wrapped  and exported the contents of `build.js` in a function called `build`.